library/std/src/time.rs

```diff
@@ -235,6 +241,12 @@ pub struct Instant(time::Instant);
 ///
 /// [`add`]: SystemTime::add
 #[derive(Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
+#[doc(alias = "insecure_time")]
+#[doc(alias = "clock_gettime")]
+#[doc(alias = "gettimeofday")]
+#[doc(alias = "SOLID_RTC_ReadTime")]
+#[doc(alias = "__wasi_clock_time_get")]
+#[doc(alias = "GetSystemTimePreciseAsFileTime")]
```

Comment:
Again, another case we have to choose which one of the functions gets to take all the time aliases.